core/domain/fs_domain.py

```diff
@@ -44,16 +53,31 @@ def size(self):
 
 
 class FileStreamWithMetadata(object):
-    """A class that wraps a file stream, but adds extra attributes to it."""
+    """A class that wraps a file stream, but adds extra attributes to it.
+
+    Attributes:
+        content: blob. The content of the file snapshots.
+        version: int. The version number of the file.
+        metadata: FileMetadata. The file metadata domain object
+    """
 
     def __init__(self, content, version, metadata):
-        """The args are a file content blob and a metadata model object."""
+        """The args are a file content blob and a metadata model object.
```

Comment:
Constructs a FileStreamWithMetadata object.

(The args are described below, you don't need to describe them here.)